rfc/0054-sdk3-management-apis.md

```diff
@@ -1494,6 +1530,192 @@ Map where top level keys are dataverse names, and values are a map of dataset na
 
     GET http://localhost:8095/analytics/node/agg/stats/remaining
 
+## CreateLink
+
+Creates a new link.
+
+### Signature
+
+```
+void CreateLink(AnalyticsLink link, [options])
+```
+
+### Parameters
+
+* Required:
+
+  * `link`: `AnalyticsLink` - the link to be created.
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+
+### Returns
+
+Nothing
+
+### Throws
+
+* `LinkAlreadyExistsException`
```

Comment:
For consistency with the existing annotations, should this be called `LinkExistsException`?

Reply:
I wonder if this should be prefixed with AnalyticsLink... since a "link" is a bit ambiguous in the wider context?